website_menu_permission/tests/test_menu_perm.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2017 Simone Orsi.
+# Copyright 2019 Therp BV <https://therp.nl>.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+import odoo.tests.common as test_common
+
+
+class TestMenuPerm(test_common.TransactionCase):
+
+    def setUp(self):
```

Comment:
please use SavepointCase and move `setUp` to `setUpClass` and replace the env like this:

```py
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True, no_reset_password=True))
```
It's going to speed up tests a lot (actually we should use this approach for all tests :man_teacher: )